libraries/core/src/main/kotlin/com/bumble/appyx/core/clienthelper/interactor/Interactor.kt

```diff
@@ -7,7 +7,7 @@ import com.bumble.appyx.core.plugin.NodeAware
 import com.bumble.appyx.core.plugin.NodeLifecycleAware
 import com.bumble.appyx.core.plugin.SavesInstanceState
 
-abstract class Interactor<N: Node>(
+open class Interactor<N: Node>(
```

Comment:
These classes don't have any abstract methods. Not sure why including the Compose detekt rules flagged these, but it doesn't hurt the API to change these.